addons/mail/models/res_users.py

```diff
@@ -80,6 +84,10 @@ def _unsubscribe_from_non_public_channels(self):
             lambda cp: cp.channel_id.public != 'public' and cp.channel_id.channel_type == 'channel'
         ).unlink()
 
+    # ------------------------------------------------------------
+    # DISCUSS
+    # ------------------------------------------------------------
```

Comment:
Instead of making these categories, I wonder at which point we should starting overriding the same model twice in the same module when each override is doing something completely different.

Reply:
Well, I was thinking about having sub-directories, like ``/models/mail`` and ``/models/discuss`` . We just have to be sure about order of imports, but could lessen noise, helps code owners and separate things without "sections" indeed.

Reply:
Yes I'd like to see that too, but I guess it is a bit too late now? I don't have time for it now (and I would rather avoid conflicts on rebase as much as possible), and once we get time for it we might as well do the discuss module split I suppose.

I'm just glad to read that you share the same idea, I'll keep in mind if the opportunity arise at some point I might just do it then.